corehq/apps/export/views/new.py

```diff
@@ -156,13 +155,31 @@ def page_context(self):
 
     @property
     def _possible_geo_properties(self):
-        if self._is_bulk_export:
+        if not toggles.SUPPORT_GEO_JSON_EXPORT.enabled(self.domain):
+            return []
+        if self._is_bulk_case_export:
             return []
 
         if self.export_type == FORM_EXPORT:
-            # 'location' is the geo-point in the form metadata
-            return ['location']
+            return self._possible_form_geo_properties
+        elif self.export_type == CASE_EXPORT:
+            return self._possible_case_geo_properties
+        return []
+
+    @property
+    def _possible_form_geo_properties(self):
```

Comment:
Hey @Charl1996 
Can you tell more about this?
Isn't the geo property a form metadata that we already know about? I am confused about why we are showing all properties from the export instance

Reply:
The user might want to export a geojson file based on GPS questions (in form builder), so we need to include those as well. They will have a doc_type set to "GeopointItem" (the meta data location also have this doc type)